src/services/ticketing-service/tests/index.test.ts

```diff
@@ -13,279 +14,376 @@ routes(router)
 app.use(bodyParser())
 app.use(router.routes())
 
-describe('routes', () => {
-  let leaseMock: Lease,
-    contactMock: Contact,
-    rentalPropertyInfoMock: RentalPropertyInfo
-  beforeEach(() => {
-    leaseMock = {
-      leaseId: '123',
-      leaseNumber: '9433',
-      leaseStartDate: new Date('2023-06-01T09:57:53.144Z'),
-      leaseEndDate: new Date('2023-06-01T09:57:53.144Z'),
-      status: 0,
-      tenantContactIds: ['P4417', 'P5602'],
-      address: undefined,
-      noticeGivenBy: undefined,
-      noticeDate: undefined,
-      noticeTimeTenant: undefined,
-      preferredMoveOutDate: undefined,
-      contractDate: undefined,
-      terminationDate: undefined,
-      lastDebitDate: undefined,
-      approvalDate: undefined,
-      tenants: [
-        {
-          contactCode: 'P4417',
-          contactKey: 'ABC',
-          firstName: 'Anneli',
-          lastName: 'Forsberg',
-          nationalRegistrationNumber: '20740522-7848',
-          birthDate: new Date('20740522'),
-          address: {
-            street: 'Gatvägen',
-            number: '29',
-            postalCode: '72489',
-            city: 'Västerås',
-          },
-          phoneNumbers: [
-            {
-              isMainNumber: true,
-              phoneNumber: '+465480978306',
-              type: 'mobile',
-            },
-            {
-              isMainNumber: false,
-              phoneNumber: '+460777174972',
-              type: 'home',
+interface OdooGetTicket {
```

Comment:
Can't you import this from where you created it in `src/services/ticketing-service/adapters/odoo-adapter.ts`? Or do they differ? 🤔